runelite-client/src/main/java/net/runelite/client/plugins/blastfurnace/BlastFurnaceConfig.java

```diff
@@ -52,4 +52,15 @@ default boolean showBarDispenser()
 	{
 		return false;
 	}
+
+	@ConfigItem(
+			keyName = "showForemanCounter",
+			name = "Show foreman fee counter",
+			description = "If under level 60 smithing, displays time left before requiring payment",
+			position = 3
+	)
+	default boolean showForemanCounter()
```

Comment:
Instead of having a config for this, display the overlay conditionally if the player is under 60 smithing.

Reply:
Resolved in 0fadcde